doc/source/ray-core/scheduling/resources.rst

```diff
@@ -128,17 +128,17 @@ There are several ways to do that depending on how you start the Ray cluster:
 Specifying Task or Actor Resource Requirements
 ----------------------------------------------
 
-Ray allows specifying a task or actor's resource requirements (e.g., CPU, GPU, and custom resources).
-The task or actor will only run on a node if there are enough required resources
+Ray allows specifying a task or actor's logical resource requirements (e.g., CPU, GPU, and custom resources).
+The task or actor will only run on a node if there are enough required logical resources
 available to execute the task or actor.
 
-By default, Ray tasks use 1 CPU resource and Ray actors use 1 CPU for scheduling and 0 CPU for running
+By default, Ray tasks use 1 logical CPU resource and Ray actors use 1 logical CPU for scheduling and 0 logical CPU for running
```

Comment:
By default, Ray tasks use 1 logical CPU resource and Ray actors use 1 logical CPU for scheduling, and 0 logical CPU for running.